app/views/staff/events/show.html.haml

```diff
@@ -103,24 +117,26 @@
               %tr
                 %td.text-primary
                   %strong CFP Closes Date:
-                - if event.closes_at.present?
+                - if event.closes_at && (event.closes_at > Time.current)
                   %td.set= link_to "Set", event_staff_edit_path
+                -elsif event.closes_at && (event.closes_at <= Time.current)
+                  %td.missing= link_to "Date has Passed", event_staff_edit_path
                 - else
                   %td.missing= link_to "Missing", event_staff_edit_path
               %tr
                 %td.text-primary
-                  %strong Session Formats:
-                - if event.session_formats.present?
-                  %td.set= link_to event.session_formats.count, '#'
+                  %strong Public Session Formats:
```

Comment:
Should these only be the public ones? Since this is the staff page, I feel like all the session formats should be displayed. It does make sense to indicate whether there are any public ones though, since opening a CFP depends on that. NM if this is something you and Marty have already discussed.